Formula/bazel.rb

```diff
@@ -17,29 +17,40 @@ class Bazel < Formula
 
   uses_from_macos "zip"
 
+  on_linux do
+    depends_on "gcc@9"
+  end
+
+  fails_with gcc: "5"
+  fails_with gcc: "6"
+  fails_with gcc: "7"
+  fails_with gcc: "8"
+
   def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
     # Force Bazel ./compile.sh to put its temporary files in the buildpath
     ENV["BAZEL_WRKDIR"] = buildpath/"work"
     # Force Bazel to use openjdk@11
-    ENV["JAVA_HOME"] = Formula["openjdk@11"].opt_libexec/"openjdk.jdk/Contents/Home"
-    ENV["EXTRA_BAZEL_ARGS"] = "--host_javabase=@local_jdk//:jdk"
-
+    ENV["JAVA_HOME"] = if OS.mac?
+      Formula["openjdk@11"].opt_libexec/"openjdk.jdk/Contents/Home"
+    else
+      Formula["openjdk@11"].opt_prefix
+    end
+    ENV["EXTRA_BAZEL_ARGS"] = "--host_javabase=@local_jdk//:jdk --verbose_failures"
+    ENV["VERBOSE"] = "yes"
     (buildpath/"sources").install buildpath.children
-
     cd "sources" do
       system "./compile.sh"
       system "./output/bazel",
              "--output_user_root",
+             "--cxxopt=-std=c++11",
              buildpath/"output_user_root",
              "build",
```

Comment:
```suggestion
             "--output_user_root",
             buildpath/"output_user_root",
             "build",
             "--cxxopt=-std=c++11",
```

[`--cxxopt`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--cxxopt) needs to go after the `build` command or else Bazel will interpret it as the value of the [`--output_user_root`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--output_user_root) option and treat `buildpath/"output_user_root"` as an invalid command.

Reply:
Adding this change to the patch.